assets/app/lib/params.rb

```diff
@@ -9,6 +9,7 @@ def self.[](key)
     end
 
     def self.add(route, k, v)
+      route = route.split('#')[0]
```

Comment:
What is happening here?

Reply:
I didn't do a good job asking my question.

Perhaps a single line of `route.split(/[\?\#]/)[0]`

Reply:
That should also work. I don't know which is preferred

Reply:
yea make it one line